services/app-web/src/pages/App/AppRoutes.tsx

```diff
@@ -101,6 +109,12 @@ const StateUserRoutes = ({
                     path={RoutesRecord.SUBMISSIONS_NEW}
                     element={<NewStateSubmissionForm />}
                 />
+                {showRateSummaryPage && (
+                    <Route
+                        path={RoutesRecord.RATES_SUMMARY}
+                        element={<RateSummary loggedInUser={loggedInUser}/>}
```

Comment:
We could also use `useAuth` hook in the `RateSummary` page component instead. Following the same pattern of not passing in props in the routes.

Reply:
Hadn't spotted that, I'll make the change, feels a lot better than passing props